crates/walrus-service/src/client/cli/args.rs

```diff
@@ -1095,6 +1096,15 @@ pub struct CommonStoreOptions {
     #[arg(long)]
     #[serde(default)]
     pub skip_tip_confirmation: bool,
+    /// Preset upload mode to tune network concurrency and bytes-in-flight.
+    #[arg(long, value_enum, default_value_t = default::upload_mode())]
+    #[serde(default = "default::upload_mode")]
+    pub upload_mode: UploadModeCli,
```

Comment:
Important: AFAIU, this currently overwrites the value from the configuration file in all cases.

Suggestion: Use an `upload_mode: Option<UploadMode>` flag, similar to what we do with similar flags like the [RPC URL](https://github.com/MystenLabs/walrus/blob/fae992e0d52888dfc50cb524dda7b0461109bca9/crates/walrus-service/src/client/cli/args.rs#L1002-L1009).

The semantics would then be to use the explicit CLI flag if it is set, use the value from the config file, or the default, in that order.

Reply:
done

Reply:
Hmm... I may have misunderstood the semantics of the `force` option. I thought it would be used to control whether or not to overwrite parameters that are explicitly set in the communication config. 

IIUC, the new version always overwrites these. As there are only two parameters, this seems reasonable to me, but we may have to revisit it if our presets control more parameters.